SConstruct

```diff
@@ -158,12 +163,12 @@ VariantDir(testBuildDirectory, pathjoin("tests", "src"), duplicate = 0)
 test = env.Program(
 	target=pathjoin("tests", "endless-sky-tests"),
 	source=RecursiveGlob("*.cpp", testBuildDirectory) + sourceLib,
-	 # Add Catch header & additional test includes to the existing search paths
+ # Add Catch header & additional test includes to the existing search paths
 	CPPPATH=(env.get('CPPPATH', []) + [pathjoin('tests', 'include')]),
 	# Do not link against the actual implementations of SDL, OpenGL, etc.
-	LIBS=sys_libs,
+	LIBS=sys_libs + ["pthread"],
```

Comment:
Just a question: Would it make sense to move pthread to the sys_libs? (I didn't check, no is also a good answer.)